### PR TITLE
fix(monitor): fix-avs-contract-addr

### PR DESCRIPTION
**Pull request by pavelnikolov**
Set AVS contract address.

task: none

### CODE REVIEW

File: e2e/types/chain.go [hunk 1]
```diff
@@ -24,7 +25,7 @@ var (
 		IsPublic:           true,
 		BlockPeriod:        15 * time.Second,
 		FinalizationStrat:  netconf.StratLatest,
-		AVSContractAddress: contracts.TestnetAVS(),
+		AVSContractAddress: common.HexToAddress("0xa7b2e7830C51728832D33421670DbBE30299fD92"),
```
Comment (corverroos) [inline e2e/types/chain.go:28]: lets remove this from `types.EVMChain` and from `netconf.Chain` and move it to `netconf.ID.Static()`.

So add `AVSContractAddress` as a field to `netconf.Static`.

**Comment by kevinhalliday** [comment]
I'm confused why this fixes it - because these are the same address. I've checked that. Any thoughts on why?

**Comment by pavelnikolov** [comment]
I'd assume that in the `monitor_only = true` manifest we don't go through some steps and that needs to be changed.

File: lib/netconf/static.go [hunk 2]
```diff
@@ -40,6 +43,10 @@ var statics = map[ID]Static{
 	Staging: {
 		OmniExecutionChainID: 16561,
 	},
+	Testnet: {
+		AVSContractAddress:   common.HexToAddress("0xa7b2e7830C51728832D33421670DbBE30299fD92"),
```
Comment (ttarsi) [inline lib/netconf/static.go:47]: does the create3-computed address not work here?

**Review by ttarsi — APPROVED** [review]

File: e2e/types/chain.go (hunk 1 above)
Comment (pavelnikolov) [reply e2e/types/chain.go]: I added just the `AVSContractAddress ` field in the testnet. I don't want to refactor the rest of the contracts now with the input EVMChain - this is the bare minimum that the Monitor service needs.

File: lib/netconf/static.go (hunk 2 above)
Comment (pavelnikolov) [reply lib/netconf/static.go:47]: It didn't work for the Monitor. But in the monitor config we don't go through all of the steps. I merge this as a workaround. This should be revisited/refactored later.

**Pull request merged**